libraries/AP_DDS/dds_xrce_profile.xml

```diff
@@ -114,6 +114,29 @@
       </historyQos>
     </topic>
   </data_writer>
+  <topic profile_name="imu__t">
+    <name>rt/ap/imu/data</name>
+    <dataType>sensor_msgs::msg::dds_::Imu_</dataType>
+    <historyQos>
+      <kind>KEEP_LAST</kind>
+      <depth>20</depth>
+    </historyQos>
+  </topic>
+  <data_writer profile_name="imu__dw">
+    <topic>
+      <kind>NO_KEY</kind>
+      <name>rt/ap/imu/data</name>
+      <dataType>sensor_msgs::msg::dds_::Imu_</dataType>
+    </topic>
+    <qos>
+      <reliability>
+        <kind>RELIABLE</kind>
```

Comment:
Can you use BEST_EFFORT reliability and VOLATILE durability because it's sensor data? The QoS you put here would have a significant overhead, especially given it's high rate.

Reply:
this change has been completed